sdk/communication/azure-communication-identity/azure/communication/identity/_generated/aio/operations/_communication_identity_operations.py

```diff
@@ -176,9 +175,6 @@ async def revoke_access_tokens(
 
         :param id: Identifier of the identity.
         :type id: str
-        :keyword api_version: Api Version. The default value is "2021-10-31-preview". Note that
```

Comment:
This is quite interesting...that the generator has removed the api_version kwarg everywhere. You know why is that? Is it because we're targeting a stable version again?

Reply:
They were moved to the [clients of the public API layer](https://github.com/Azure/azure-sdk-for-python/blob/bb27e0594f1bf2c15dac655598cc615c3b194d8d/sdk/communication/azure-communication-identity/azure/communication/identity/_generated/_communication_identity_client.py#L35) which are the main interaction points for 
the convenience layer

Reply:
nice